cmd/sync/sync.go

```diff
@@ -54,4 +74,22 @@ func main() {
 	if err := sync(); err != nil {
 		panic(err)
 	}
+	if *once {
+		return
+	}
+
+	sig := make(chan os.Signal, 1)
+	signal.Notify(sig, os.Interrupt, syscall.SIGTERM)
+	for {
+		select {
+		// TODO: Potentially expose an endpoint to request syncs.
+		// on top of this.
+		case <-time.After(3 * time.Minute):
+			if err := sync(); err != nil {
```

Comment:
your loop's backwards - shouldn't need `err := sync()` outside the loop and inside it ;)

Reply:
It's required for `run--once` to work. Otherwise I need to keep an index inside the loop.

Reply:
```
for {
sync()
if *once {return}
<-time.After()
}
```
?

Reply:
sure